jwe/internal/cmd/genheader/main.go

```diff
@@ -3,14 +3,15 @@ package main
 import (
 	"bytes"
 	"fmt"
-	"go/format"
 	"log"
 	"os"
 	"sort"
 	"strconv"
 	"strings"
 
+	"github.com/lestrrat-go/jwx/jwe"
```

Comment:
no, don't import from the package that we're generating into. I know this causes problems